submissions/api.py

```diff
@@ -413,6 +413,65 @@ def get_all_submissions(course_id, item_id, item_type, read_replica=True):
         yield data
 
 
+def get_all_course_submission_information(course_id, item_type, read_replica=True):
+    """ For the given course, get all student items of the given item type, all the submissions for those itemes,
+    and the latest scores for each item. If a submission was given a score that is not the latest score for the
+    relevant student item, that score will be excluded from the results.
```

Comment:
Maybe: ... If a submission was given a score that is not the latest score, it will still be included but without a score.